bears/general/InvalidLinkBear.py

```diff
@@ -44,7 +44,8 @@ def find_links_in_file(file, timeout, link_ignore_regex):
         link_ignore_regex = re.compile(link_ignore_regex)
         regex = re.compile(
             r'((ftp|http)s?://[^.:%\s_/?#[\]@\\]+\.(?:[^\s()%\'"`<>|\\]+|'
-            r'\([^\s()%\'"`<>|\\]*\))*)(?<!\.)(?<!,)')
+            r'\\]+|\([^\s()\'\"`<>|\\]*\))%?[0-9]*(?:[^\s()%\'\"`<>|\\]+|'
+            r'\([^\s()\'\"`<>|\\]*\))*)(?<!\.)(?<!,)')
```

Comment:
I tried different things and only this seemed to work, Looks like I will have to look for another solution :/